lib/elixir/lib/function.ex

```diff
@@ -166,4 +166,27 @@ defmodule Function do
   @doc since: "1.7.0"
   @spec info(fun, item) :: {item, term} when item: information
   def info(fun, item), do: :erlang.fun_info(fun, item)
+
+  @doc """
+  Returns its input `value`, which can be given as an anonymous function
+  to transformation functions.
+
+  ## Examples
+
+      iex> Function.identity("Hello world!")
+      "Hello world!"
+
+      iex> 'abcdaabccc' |> Enum.sort |> Enum.chunk_by(&Function.identity/1)
```

Comment:
```suggestion
      iex> 'abcdaabccc' |> Enum.sort() |> Enum.chunk_by(&Function.identity/1)
```
:nail_care: I think we could use in the example a code following the default rules from the formatter.